cluster/extensions-javadsl/src/main/java/com/lightbend/lagom/javadsl/cluster/projections/Projections.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.javadsl.cluster.projections;
+
+import akka.annotation.ApiMayChange;
+
+import java.util.concurrent.CompletionStage;
+
+/** TODO: docs */
+@ApiMayChange
+public interface Projections {
+
+  CompletionStage<DesiredStatus> getStatus();
```

Comment:
`getDesiredState`

I'm also wondering if `Desired` is a good suffix. Now that I'm looking more at it, I find it weird. What about `RequestedState`?

Reply:
I think the name will still evolve when we introduce the idea of `initialState`. I've renamed (locally) to `DesiredStatus` and would rather freeze further renames for the time being.